packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -320,12 +318,11 @@ export default {
                         `Content Cards (metadataContentCards) - Successfully received content cards. Key: (${this.loggingKey})`,
                         this.$store,
                         {
-                            tags: this.tags,
+                            ...this.defaultLoggingData,
+                            Count: cards.length,
                             data: {
-                                key: this.loggingKey,
                                 source: CARDSOURCE_METADATA
-                            },
```

Comment:
Does this merge cleanly into the destructured `this.defaultLoggingData`, or simply replace the `data` value with one that doesn't include the key? I'd suspect it does the latter, unless you do something like:
```suggestion
                            data: {
                                ...this.loggingKey.data,
                                source: CARDSOURCE_METADATA
```

Reply:
indeed my mistake

Reply:
fixed :)